index.d.ts

```diff
@@ -2837,7 +2838,7 @@ declare namespace Eris {
     editRole(guildID: string, roleID: string, options: RoleOptions, reason?: string): Promise<Role>; // TODO not all options are available?
     editRoleConnectionMetadataRecords(data: ApplicationRoleConnectionMetadata[]): Promise<ApplicationRoleConnectionMetadata[]>;
     editRolePosition(guildID: string, roleID: string, position: number): Promise<void>;
-    editSelf(options: { avatar?: string; username?: string }): Promise<ExtendedUser>;
+    editSelf(options: { avatar?: string; banner?: string; username?: string }): Promise<ExtendedUser>;
```

Comment:
thonk why wasn't this using EditSelfOptions in the first place

Reply:
Well, somehow I didn't notice that, but there you go.